Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.3'
+ruby '2.2.4'
```

Comment:
It's good practice to specify the ruby version in the gemfile and probably also the `.ruby-version` file, above. 2.2.4 is the latest version in the 2.2.x branch and it contains a lot of bug fixes.